modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -189,6 +189,14 @@ config TFM_IMAGE_VERSION_NS
 	  versions of both the secure firmware and non-secure firmware. This sets
 	  the non-secure firmware's version for rollback prevention.
 
```

Comment:
Can you update the help text for TFM_FLASH_MERGED_BINARY to mention BL1 as well?